packages/core/contracts/bot-helpers/uniswap-broker/UniswapBrokerV3.sol

```diff
@@ -0,0 +1,631 @@
+pragma solidity ^0.8.0;
+pragma abicoder v2;
+
+import "@uniswap/lib/contracts/libraries/TransferHelper.sol";
+
+import "@uniswap/v3-core/contracts/interfaces/IUniswapV3Pool.sol";
+
+import "@uniswap/v3-periphery/contracts/interfaces/ISwapRouter.sol";
+
+import "@uniswap/v3-core/contracts/libraries/LowGasSafeMath.sol";
+import "@uniswap/v3-core/contracts/libraries/LiquidityMath.sol";
+import "@uniswap/v3-core/contracts/libraries/BitMath.sol";
+import "@uniswap/v3-core/contracts/libraries/UnsafeMath.sol";
+import "@uniswap/v3-core/contracts/libraries/SafeCast.sol";
+
+/**
+ * @title UniswapBrokerV3
+ * @notice Trading contract used to arb uniswapV3 pairs to a desired "true" price. Intended use is to arb UMA perpetual
+ * synthetics that trade off peg. This implementation can ber used in conjunction with a DSProxy contract to atomically
+ * swap and move a uniswap market.
+ */
+
+contract UniswapBrokerV3 {
+    using SafeCast for uint256;
+    using LowGasSafeMath for uint256;
+    using LowGasSafeMath for int256;
```

Comment:
we _could_ do that but right now everything in this file is using the `mul`/`div` syntax. dropping this would require a larger refactor. I'd say that for consistency with our other contract let's keep it in and then we can do a larger refactor where we remove all of this tuff in one go.

Reply:
That makes sense. A one-time total refactor probably would be better.

As a slightly less invasive suggestion, could we use OZ's safe math implementation? I only suggest it bc it's cheaper gas-wise and more consistent with our other code.